docs/spec-find.md

```diff
@@ -73,7 +73,7 @@ enum SortDirectionEnum {
 ```
 
 To enable filtering for specific Scalars we can create individual InputTypes.
-For example for String scalar this could be:
+For example, for the five built-in Scalars this could be:
```

Comment:
Reworded this slightly since we mention the String scalar here but then go on to list 3 others